contrib/grpcplugins/action/archive/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/mholt/archiver"
```

Comment:
The dependency is not in cds vendors, this can't compile

Reply:
We will add it after the merge because of the go 1.11 PR

Reply:
Yes I didn't add it in the vendors.